### ISSUE TITLE
Sprint 4 – Scrum Meeting (21-12-2016)

### ISSUE THREAD

**Issue by lkc442012** [mewind/RSDSupermarketPOS#23]
Good morning everyone! Today is the third day of Sprint 4, may I know the progression until now?

**Comment by huylin25** [comment]
I have finished the generation of sales report on yesterday. I will upload the code later.

**Comment by lkc442012** [comment]
I heard that you are facing some problem on that, do you fix it?

**Comment by huylin25** [comment]
Yes, thanks to Sin Nee and Chun Xiong. They helped me a lot so that I can finished it on time.

**Comment by juzplayplay** [comment]
You are welcome! That's part of my job.

**Comment by mewind** [comment]
Its ok!

**Comment by huylin25** [comment]
I have successfully connect the report with the database using SQL command. The system now is able to display the product name, price, quantity and total sales in the sales report page.

**Comment by lkc442012** [comment]
Great! So we can proceed to the exception report?

**Comment by juzplayplay** [comment]
Yes, actually I am doing the generation of exception report now. Yesterday, Sin Nee and I helped Huy Lin to solve her problems. This give me a clear idea on what should be do in my part. By refering to Huy Lin's code, I think I am able to finish the exception report by tomorrow.

**Comment by mewind** [comment]
What do you want to include in the exception report?

**Comment by juzplayplay** [comment]
In my opinion, I think the report should display the product name, product category and total quantity. These information is needed to know the least sales of the product in a month.

**Comment by mewind** [comment]
Okay, it makes sense.

**Comment by lkc442012** [comment]
It seems like you already have a very clear idea to develop it.

**Comment by juzplayplay** [comment]
I will try my best to finish it as soon as possible.

**Comment by huylin25** [comment]
If you have any question you can ask me, since I have already finished my part. I may give you some suggestions or solutions on solving the problem.

**Comment by juzplayplay** [comment]
Thanks!

**Comment by lkc442012** [comment]
Okay. We are waiting for your good news!

**Comment by lkc442012** [comment]
We can dismiss the meeting now since there is not much thing to discuss on this week.

**Comment by mewind** [comment]
Thank you Kar Cheng for conducting the meeting!
